CardPayments/src/main/java/com/paypal/android/cardpayments/CardClient.kt

```diff
@@ -65,6 +67,15 @@ class CardClient internal constructor(
     }
 
     private suspend fun confirmPaymentSource(activity: FragmentActivity, cardRequest: CardRequest) {
+        try {
+            cardAPI.fetchCachedOrRemoteClientID()
+        } catch (e: PayPalSDKError) {
```

Comment:
General Kotlin/Android question here, but is `e` vs more explicit `error` the best practice for the language/platform? I'm still digging into Kotlin as a whole so am not totally sure what the language practices are.

Reply:
I'm not sure what best practice is, but we use `e` in the other try-catch blocks in the SDK, so I went with this for consistency. [Kotlin docs examples](https://kotlinlang.org/docs/exceptions.html) also use `e`